pkg/agent/profiler/profiler.go

```diff
@@ -20,12 +20,17 @@ var (
 	ProfileInuseSpace   = spy.ProfileInuseSpace
 )
 
+const (
+	DefaultSampleRate = 100 // 100 times per second
+)
+
 var DefaultProfileTypes = []ProfileType{ProfileCPU, ProfileAllocObjects, ProfileAllocSpace, ProfileInuseObjects, ProfileInuseSpace}
 
 type Config struct {
 	ApplicationName string // e.g backend.purchases
 	ServerAddress   string // e.g http://pyroscope.services.internal:4040
 	AuthToken       string // specify this token when using pyroscope cloud
+	SampleRate      uint32
```

Comment:
```suggestion
	# SampleRate      uint32
```

I think we should comment this out until we implement golang profiler support (https://github.com/pyroscope-io/pyroscope/issues/191)